qemux86-64.robot

```diff
@@ -33,6 +33,14 @@ Check host OS fingerprint file
   Verify resin-root.fingerprint in "${path_to_fingerprint}"
   Unmount "${mount_destination}"
   Detach loop device "${path_to_loop}"
+Get host OS version of the image
+  Set up loop device for "${image}"
```

Comment:
Coming to think of it - should we merge 

Set up loop device for "${image}" and Find the last loop device that was assigned to one keyword which has a behavior as follows

${LOOPDEVICE} = Set up loop device for "${image}"